services/workflows-service/src/errors.ts

```diff
@@ -25,24 +26,55 @@ export class SessionExpiredException extends common.UnauthorizedException {
   message!: string;
 }
 
-export class AjvValidationError extends common.BadRequestException {
+class DetailedValidationError {
+  @ApiProperty()
+  message!: string;
+
+  @ApiProperty()
+  path!: string;
+}
+
+export class ValidationError extends common.BadRequestException {
   @ApiProperty()
   statusCode!: number;
   @ApiProperty()
-  message!: string;
+  message: string = 'Validation error';
```

Comment:
Let's match the casing with "Bad Request" and the rest.
```suggestion
  message: string = 'Validation Error';
```

Reply:
its a message should have

Reply:
Is that true for a majority of messages in our system? Having in mind the conclusion of the errors thread and meeting we had.